### PR TITLE
 add oracle e2e test script

### PR DESCRIPTION
**Pull request by zzzk1**
[Task] e2e test script for oracle #5302 

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by moremind** [comment]
not right

**Comment by moremind** [comment]
pls add right script and yaml

**Comment by zzzk1** [comment]
Okay, I think I know how to do this

File: shenyu-e2e/shenyu-e2e-case/k8s/shenyu-oracle.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    matchLabels:
+      app: shenyu-oracle
+  template:
+    metadata:
+      labels:
+        app: shenyu-oracle
+    spec:
+      containers:
+        - name: shenyu-oracle
+          image: registry.aliyuncs.com/helowin/oracle_11g
```
Comment (moremind) [inline shenyu-e2e/shenyu-e2e-case/k8s/shenyu-oracle.yml:35]: use docker hub images?

File: shenyu-e2e/script/storage_init_oracle.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+mkdir -p /tmp/shenyu-e2e/oracle
+mkdir -p /tmp/shenyu-e2e/driver/oracle
```
Comment (moremind) [inline shenyu-e2e/script/storage_init_oracle.sh:2]: add license header

**Review by moremind — CHANGES_REQUESTED** [review]

**Comment by moremind** [comment]
may you refer to: #5309

**Pull request closed without merging**